### PR TITLE
doc: Update "Enable S5 in ACRN" tutorial

### PR DESCRIPTION
**Pull request by gvancuts**
Update the "Enable S5 in ACRN" tutorial to:
* Make the language clearer at places
* Consistent use of "Lifecycle Manager" and "life_mngr"
* Change "ACK" to "ACKED"
* Use a ReST reference to point at the architectural diagram
* Update build instructions

Signed-off-by: Geoffroy Van Cutsem <geoffroy.vancutsem@intel.com>

### CODE REVIEW

File: doc/tutorials/enable_s5.rst
Comment (gvancuts) [inline doc/tutorials/enable_s5.rst:66]: @NanlinXie I really wish these error messages would be more indicative of the actual problem. Can we get that fixed in the code?

File: doc/tutorials/enable_s5.rst
Comment (gvancuts) [inline doc/tutorials/enable_s5.rst:99]: I don't thing the lines we want to highlight still appear in those scripts... @NanlinXie , can ask one of the developer to look at this?

File: doc/tutorials/enable_s5.rst
Comment (gvancuts) [inline doc/tutorials/enable_s5.rst:100]: Same for this script...

**Comment by dbkinder** [comment]
@NanlinXie @gvancuts Can we get a resolution for this PR's comments...

**Comment by gvancuts** [comment]
This is also linked to #5936. We need to improve our documentation for this component, and possibly our design too.

**Comment by gvancuts** [comment]
On a sidenote: I think it would be better to not trigger all the checks until the Pull Request has been moved from "draft" to "ready for review". Is this possible?

**Comment by dbkinder** [comment]
@gvancuts @NanlinXie Is this doc update going to happen for the 2.5 release?

**Comment by dbkinder** [comment]
@gvancuts Ping, in case you forgot about this PR :)

**Comment by gvancuts** [comment]
> @gvancuts Ping, in case you forgot about this PR :)

I have not... but no bandwidth to go finish it yet :-(

**Comment by dbkinder** [comment]
Hey @gvancuts What should we do with this PR?

**Comment by NanlinXie** [comment]
We plan to refresh S5 document in v2.7 together.

**Comment by gvancuts** [comment]
Not quite as complete as I wanted it, but this is as good as it gets. It will need to be further updated once the new design lands anyway.

File: doc/tutorials/enable_s5.rst
Comment (dbkinder) [inline doc/tutorials/enable_s5.rst:61]: don't need the double quote around this

File: doc/tutorials/enable_s5.rst
Comment (dbkinder) [inline doc/tutorials/enable_s5.rst:75]: Let's not mix the terms "power off" and "shut(ting) down".  I think "shut(ting) down" might be the more appropriate term. 
```suggestion
shut down successfully or not, and decides when to power the Service VM itself off.
```

File: doc/tutorials/enable_s5.rst
Comment (dbkinder) [inline doc/tutorials/enable_s5.rst:80]: I'd delete "current" since all the documentation is talking about the current design and we generally avoid talking about what we may do in the future (no promises).

File: doc/tutorials/enable_s5.rst
Comment (dbkinder) [inline doc/tutorials/enable_s5.rst:83]: same comment here, "shutdown" vs. "power off" in this paragraph.  Recommend changing to use "shut down" instead of "power off" on the next line,

File: doc/tutorials/enable_s5.rst
Comment (dbkinder) [inline doc/tutorials/enable_s5.rst:113]: is that trailing space inside the double quote needed?

**Review by dbkinder — CHANGES_REQUESTED** [review]

**Comment by dbkinder** [comment]
@NanlinXie @wenlingz How do we know if this "YOCTO/ADL/INDUSTRY/WaaG_PREEMPT_RT_Test — Failed" can be ignored because it's something like a CI networking error, as has happened before, or when it really is a failure and we shouldn't merge this PR?

File: doc/tutorials/enable_s5.rst
Comment (fitchbe) [inline doc/tutorials/enable_s5.rst:74]: "check if" --> "check whether"

File: doc/tutorials/enable_s5.rst
Comment (fitchbe) [reply doc/tutorials/enable_s5.rst:75]: It is the Service VM's responsibility to check...and **to decide**... (not "decides").

**Review by fitchbe — APPROVED** [review]

File: doc/tutorials/enable_s5.rst
Comment (gvancuts) [reply doc/tutorials/enable_s5.rst:113]: I don't think so (not tested it though ;-)... I think it's more of an extra safety measure. I'll remove them and also put each of those arguments on a separate line, that should take care of the need to separate them.

**Comment by gvancuts** [comment]
Thanks for your comments @dbkinder and @fitchbe , I believe I have addressed them all. PTAL

**Review by dbkinder — APPROVED** [review]
LGTM, thanks.

**Pull request merged**
